packages/builder/src/pages/builder/portal/account/auditLogs/index.svelte

```diff
@@ -86,8 +86,15 @@
   $: userPage = $userPageInfo.page
   $: logsPage = $logsPageInfo.page
 
+  $: userList = {
+    ...(userList || {}),
+    ...$users.data?.reduce((p, c) => {
+      p[c._id] = c
+      return p
+    }, {}),
+  }
   $: sortedUsers = sort(
-    enrich($users.data || [], selectedUsers, "_id"),
+    enrich(Object.values(userList), selectedUsers, "_id"),
```

Comment:
Potential null pointer - can we be certain userList will always be an array here? Could do `userList || []` to be safe?

Reply:
This will never be null, as we are setting it up on the line above with default values already